src/opnsense/service/templates/OPNsense/Unbound/core/dot.conf

```diff
@@ -29,26 +29,34 @@ forward-zone:
 {%    for forward in forwards %}
   forward-addr: {{ forward.server }}{% if forward.port %}@{{ forward.port }}{% endif %}
 {%    set domain_opts.forward_tcp_upstream = domain_opts.forward_tcp_upstream or forward.forward_tcp_upstream == '1' %}
+{%    set domain_opts.forward_first = domain_opts.forward_first or forward.forward_first == '1' %}
```

Comment:
best initialize to False in `namespace()` for consistency

Reply:
@planetf1 no problem, GitHub doesn't let you comment on unchanged lines, but my proposed change is in line 26:

```
{% set domain_opts = namespace(forward_tcp_upstream=False) %}
```

should probably be changed to the following for consistency:
 
```
{% set domain_opts = namespace(forward_tcp_upstream=False, forward_first=False) %}
```

Reply:
@AdSchellevis thanks for explaining!

Updated, checked locally (I needed to remove .mount_done). rebased & pushed.